python311/Dockerfile

```
@@ -1,9 +1,9 @@
-FROM debian:bookworm-slim
+FROM python:3.11.13-slim-bookworm
```

```suggestion
FROM python:3.11.12-slim-bookworm
```
don't misread the 3.11 changelog :laughing: